tests/parser/TuningTests.cpp

```diff
@@ -378,4 +398,16 @@ BOOST_AUTO_TEST_CASE(TuningTest) {
       BOOST_CHECK_EQUAL(NEWTMX, 13);     
 
   }
+  /*** TIMESTEP 11 ***/
+  {
+      std::size_t timestep = 11;
+      BOOST_CHECK_CLOSE(schedule[timestep].max_next_tstep(true), 1.0 * Metric::Time, diff);
+      BOOST_CHECK_CLOSE(schedule[timestep].max_next_tstep(false), 1.0 * Metric::Time, diff);
+  }
+  /*** TIMESTEP 12 ***/
+  {
+      std::size_t timestep = 12;
+      BOOST_CHECK_CLOSE(schedule[timestep].max_next_tstep(true), 1.0 * Metric::Time, diff);
+      BOOST_CHECK_CLOSE(schedule[timestep].max_next_tstep(false), 2.0 * Metric::Time, diff);
+  } 
```

Comment:
Very good.  Do you think you could add a test for the `max_next_tstep() < 0` case as well&ndash;i.e., for when there is no `NEXTSTEP` and we do not enable `TUNING`?